includes/class-hours.php

```diff
@@ -150,7 +168,12 @@ protected function get_fields( array $instance, array $fields = [], $ordered = t
 				'description' => __( 'Enter your hours in the following fields.', 'contact-widgets' ),
 			],
 			'additional_content' => [
-
+				'label'       => __( 'Additional Info.', 'contact-widgets' ),
+				'type'          => 'textarea',
+				'sanitizer'     => function( $value ) { return current_user_can( 'unfiltered_html' ) ? $value : wp_kses_post( stripslashes( $value ) ); },
+				'escaper'       => function( $value ) { return nl2br( apply_filters( 'widget_text', $value ) ); },
+				'form_callback' => 'render_form_textarea',
+				'description' => __( 'Enter additional information about your store.', 'contact-widgets' ),
```

Comment:
@EvanHerman I think `Enter additional information about your business.` might be more universal.